metadata-ingestion/tests/unit/test_transform_dataset.py

```diff
@@ -287,11 +274,8 @@ def test_simple_dataset_ownership_transformation(mock_time):
         ]
     )
 
-    # Verify that the third entry is unchanged.
-    assert inputs[2] == outputs[2].record
-
     # Verify that the last entry is EndOfStream
-    assert inputs[3] == outputs[4].record
+    assert inputs[-1] == outputs[-1].record
```

Comment:
Smart change, not sure why are we testing it at all though, shouldn't it be true for every transformer?

Reply:
No harm in testing it, right?